### PR TITLE
Use SSML for voice messages (LG-4675)

### PR DESCRIPTION
**Pull request by zachmargolis**
**Why**: To give more control over timing and pauses between
parts of OTPs

See sample recordings in Slack: https://gsa-tts.slack.com/archives/CEUQ9FXNJ/p1622662395035200

### CODE REVIEW

File: lib/telephony/configuration.rb [hunk 1]
```diff
@@ -50,6 +50,8 @@ class Configuration
     attr_writer :adapter
     attr_reader :pinpoint
     attr_accessor :logger
+    attr_accessor :voice_pause_time
+    attr_accessor :voice_rate
```
Comment (aduth) [inline lib/telephony/configuration.rb:54]: Should we have defaults for these?

**Review by aduth — APPROVED** [review]
Nice 👍

File: lib/telephony/configuration.rb (hunk 1 above)
Comment (zachmargolis) [reply lib/telephony/configuration.rb:54]: channelling my inner @pauldoomgov.... no, because hardcoded defaults end up being bad

Comment (aduth) [reply lib/telephony/configuration.rb:54]: Fair enough 👍

**Review by mitchellhenke — APPROVED** [review]

**Pull request merged**